Dynamic/js/LivelyProperties.js

```diff
@@ -102,7 +104,36 @@ function livelyPropertyListener(name, val) {
     case "pixel-noise":
       pixelNoise.classList.toggle("hide", !val);
       break;
-
+    case "visualiser-toggle":
+      visualiser.classList.toggle("hide", !val);
+      break;
+    case "show-when-not-playing":
+      visualiser.classList.toggle("show-when-not-playing", !val);
+      break;
+    case "sorting-mode":
+      switch (val) {
+        case 0:
+          sortSoundArray = concentratedSort;
+          numDots = 128;
```

Comment:
Why is numDots lower only for concentratedSort?

Reply:
Oh, the original was like that.

Reply:
Yes I just copy pasted most of the code :)

Reply:
I should probably include the license one sec...